### ISSUE TITLE
select-box()

### ISSUE THREAD

**Issue by dbox** [jescalan/axis#207]
We should add basic styling for `select` dropdowns ala http://bavotasan.com/2011/style-select-box-using-only-css/ (obviously one that looks better, but the basic code premise is right.)

I'll post codepen example soon.

**Comment by dbox** [comment]
https://github.com/filamentgroup/select-css

**Comment by jenius** [comment]
+1 for this. Second example looks great

**Comment by dbox** [comment]
Need to do some more customization and testing but I have basic mixin going here:

http://codepen.io/dbox/pen/ZGZzmW?editors=110

Best part is you can pass a custom dropdown image or, if not it will default to using Axis' triangle. :)

**Comment by jenius** [comment]
This is lovely. Nice and clean and basic, just how we need it :+1:

**Comment by dbox** [comment]
Well the alt image WAS working .... May end up being too problematic. but I"ll keep tinkering

**Comment by dbox** [comment]
Ah, they were just preventing hotlinking.

It does bring up the issue though, if we're goin to do custom image option, user will have to pass image height/width :/

**Comment by jenius** [comment]
Sweet! PR dat :+1:

**Comment by dbox** [comment]
I just have to come up with one little fix so that borders look better with dark backgrounds, then i will!

**Comment by dbox** [comment]
Okay this thing just got bonkers but it will be insanely handy:

http://codepen.io/dbox/pen/ZGZzmW?editors=110

Don't get too freaked out about # of arguments. Most use cases could be done just with `$border-radius` and `background-color`. It uses the same logic as `button()` to dynamically change font-size and a few other attributes based on `background-color`.

Sample usage:

```stylus
div.custom-select
  select-box()
```

```stylus
div.custom-select
  select-box(0, blue)
```

**Comment by dbox** [comment]
Pretty much ready to go. Here's a better demo (pretend like it's stylus)

http://s.codepen.io/dbox/debug/xwNMZd

**Comment by jenius** [comment]
:+1:

**Comment by dbox** [comment]
available in 2.0
